tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -32,11 +32,21 @@ std::uint64_t GetNewModelId()
 }
 } // anonymous namespace
 
+ROOT::Experimental::RFieldZero &ROOT::Experimental::Internal::GetFieldZero(ROOT::Experimental::RNTupleModel &model)
+{
+   return *model.fFieldZero;
```

Comment:
Similar comment to https://github.com/root-project/root/pull/16714/files#r1806707632